server/src/main/java/org/apache/cassandra/sidecar/config/ServiceConfiguration.java

```diff
@@ -92,7 +92,7 @@ default List<SocketAddress> listenSocketAddresses()
     /**
      * @return the maximum time skew allowed between the server and the client
      */
-    int allowableSkewInMinutes();
+    SecondBoundConfiguration allowableTimeSkew();
```

Comment:
Is it ok to be using `SecondBoundConfiguration` instead of `MinuteBoundConfiguration` for this one?

Reply:
we don't have a `MinuteBoundConfiguration` class, and the reason I avoided adding one was because it feels to me the time skew resolution is too coarse. This resolution should have been at the second level to begin with IMO. But I am hoping that we think about why the resolution is that coarse and maybe reduce it to seconds.

Reply:
It should be minutes btw... this isn't a replacement for NTP, it's a guardrail to make sure that the Spark workers aren't so far skewed from C* that deletes and TTLs do weird things (since Spark writes the row timestamps in the Analytics library, where we consume this endpoint).  It is both possible and acceptable for there to be several minutes of skew in environments that don't have tightly controlled NTP sync, and specifying this in seconds really doesn't make any sense (is 65 seconds acceptable, but 66 not?)

That said, I don't really mind that it's _set_ in seconds, but the expectation that somehow the resolution of seconds is meaningful here isn't realistic.

Reply:
thanks for the context. I think we can have a minute bound configuration if that's the case.

Reply:
addressed this